.gitignore

```diff
@@ -9,7 +9,7 @@ proxies/*.yml
 temp
 shield
 !proto/*
-.shield.yaml
+config.yaml
```

Comment:
```suggestion
/config.yaml
```
@mabdh Without this it will ignore any config.yaml file anywhere.

Reply:
yes you are correct, thanks @ravisuhag for the heads up. 
will recheck what else missing